extensions/resource-deployment/src/utils.ts

```diff
@@ -30,12 +31,18 @@ export function setEnvironmentVariablesForInstallPaths(tools: ITool[]): void {
 			// construct an env variable name with NoteBookEnvironmentVariablePrefix prefix
 			// and tool.name as suffix, making sure of using all uppercase characters and only _ as separator
 			const envVarName = getRuntimeBinaryPathEnvironmentVariableName(t.name);
-			process.env[envVarName] = t.installationPathOrAdditionalInformation;
+			env[envVarName] = t.installationPathOrAdditionalInformation;
 			installationPaths.add(path.dirname(t.installationPathOrAdditionalInformation));
 		}
 	});
 	if (installationPaths.size > 0) {
 		const envVarToolsInstallationPath: string = [...installationPaths.values()].join(path.delimiter);
-		process.env[ToolsInstallPath] = envVarToolsInstallationPath;
+		env[ToolsInstallPath] = envVarToolsInstallationPath;
+	}
+}
+
+export function assert(condition: any, message?: string): asserts condition {
```

Comment:
Why not have condition be a boolean?

Reply:
It was to allow a truthy expression. Will change.

Reply:
fixed.